### PR TITLE
(DOCSP-17228): BI Connector quick wins

### PR DESCRIPTION
**Pull request by jwilliams-mongo**
Based on https://docs.google.com/document/d/1g5nLcfk2cmLIMK2Tj9huzItU4ASxzMGNtOiW4doFnnU/edit# updated the BI Connector docs.

[JIRA](https://jira.mongodb.org/browse/DOCSP-17228)

Stage:

- [What is the MongoDB Connector for BI](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-17228/what-is-the-bi-connector/) updated images and text in system components section
- [Quick Start Guide for Windows](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-17228/local-quickstart/) updated based on guidance
- [JDBC Driver](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-17228/reference/jdbc-driver/) added new page
- [Connect BI Tools](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-17228/client-applications/) updated based on guidance (see questions about this on the Files tab)
- [Connect from Microsoft Power BI Desktop](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-17228/connect/powerbi/) updated link

Follow up Q: this is for 2.14 only, correct? @sposetti

### CODE REVIEW

File: source/client-applications.txt [hunk 1]
```diff
@@ -4,102 +4,20 @@ Connect BI Tools
...
-
-   .. card::
-      :cta: Looker
-      :url: https://docs.mongodb.com/bi-connector/current/connect/looker
-      :icon: /images/bi-connector/icons/looker-logo.png
-      :icon-alt: Looker icon
+After you install and configure the |bi-short|, you can use |odbc| and
+|jdbc| drivers to connect from many BI Tools and clients. The sections
+below provide basic instructions for connecting to |bi-short| using each
+tool.
```
Comment (jwilliams-mongo) [inline source/client-applications.txt:10]: based on the doc it seems as if @sposetti wanted to remove all other tools from this page. Is this correct? If so I can delete the source files for these too.

Comment (sposetti) [reply source/client-applications.txt:10]: Correct. Wanted to shrink the list of tools that we doc.

**Comment by sposetti** [comment]
Overall, LGTM. Thanks!

File: source/what-is-the-bi-connector.txt [hunk 2]
```diff
@@ -48,27 +48,27 @@ BI Connector
...
+The following sections describe possible configurations for a
 BI system.
 
 Hosted Database and |bi-short|
 ------------------------------
 
 .. note::
 
-   If you don’t want to install and manage the |bi-short|, you can use
+   If you don't want to install and manage the |bi-short|, you can use
```
Comment (jmd-mongo) [inline source/what-is-the-bi-connector.txt:71]: Nice catch!

**Review by jmd-mongo — COMMENTED** [review]
Hi JW,

Overall, LGTM!

Thanks,
Joe

**Pull request merged**